unit/test_tags.py

```diff
@@ -11,6 +11,8 @@
 
 class TestTags(unittest.TestCase):
     def setUp(self):
+        self.maxDiff = None
```

Comment:
What are you using this code for?

Reply:
Because if lists comparison fails, pytest doesn't show full diff, but shows warning: `Diff is 664 characters long. Set self.maxDiff to None to see it.`. I used this flag to see why did tests fail.